setup.py

```diff
@@ -1,23 +1,149 @@
-from setuptools import setup, find_packages
 from codecs import open
 from os import path
+from setuptools import setup, find_packages
+from platform import python_version, uname, mac_ver
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from README.md file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+# Get release version
+with open(path.join(here, 'ivadomed', 'version.txt')) as f:
+    version = f.read().strip()
 
 # Manually specified, more generic version of the software.
 # See: https://stackoverflow.com/a/49684835
-with open('requirements_common.txt') as f:
+with open(path.join(here, 'requirements.txt'), encoding='utf-8') as f:
     requirements = f.readlines()
 
-# Get README
-this_directory = path.abspath(path.dirname(__file__))
-with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
-    long_description = f.read()
 
-# Get Release version
-path_version = path.join(this_directory, 'ivadomed', 'version.txt')
-with open(path_version) as f:
-    version = f.read().strip()
+def get_whl_info():
+    """Returns the whl info to handpick torch whls.
+
+    whl format: {dist}-{version}(-{build})?-{python_tag}-{abi_tag}-{platform}.whl
+
+    Returns:
+        str, str, str: python_tag, abi_tag, platform.
+    """
+    py_version = python_version()[0:3].replace('.', '')
+    python_tag = f'cp{py_version}'
+
+    if py_version == '36' or py_version == '37':
+        abi_tag = f'{python_tag}m'
+    elif py_version == '38' or py_version == '39':
+        abi_tag = python_tag
+
+    sysinfo = uname()
+    if sysinfo.system == 'Linux':
+        if sysinfo.machine == 'x86_64':
+            platform = 'linux_x86_64'
+            return python_tag, abi_tag, platform
+        elif sysinfo.machine == 'aarch64':
+            raise ValueError(f"ivadomed doesn't handpick torch whls for {sysinfo.machine}, \
+            install torch==1.8.0+cpu, torchvision==0.9.0+cpu or \
+            torch==1.8.1+cu111, torchvision==0.9.1+cu111 from \
+            https://pytorch.org/get-started/previous-versions/")
+    elif sysinfo.system == 'Windows' and sysinfo.machine == 'AMD64':
+        platform = 'win_amd64'
+        return python_tag, abi_tag, platform
```

Comment:
Maybe we should raise on x86? I don't have a 32bit windows to check this though :(. We can just say maybe 32bit not supported? Don't need to get into the details.

Reply:
Lol even I didn't know about it but "PyTorch doesn’t work on 32-bit system. Please use Windows and Python 64-bit version." as per [official PyTorch FAQs](https://pytorch.org/docs/stable/notes/windows.html). Since, `sysinfo = uname()` and `uname()` still returns `AMD64` on a 64-bit machine with intel processor. Moreover, I couldn't find the output of `uname()` for 32-bit Windows machine by Googling. What about when someone raises an issue regarding this, we then update?